server/chat-commands/info.ts

```diff
@@ -2509,14 +2509,14 @@ export const commands: ChatCommands = {
 				return this.errorReply('Invalid image');
 			}
 		}
-		buf += Utils.html`<br /><p style="margin-left:5px;font-size:9pt;color:gray"><small>(Requested by ${request.name})</small>`;
+		buf += Utils.html`<br /><div class="infobox"><small>(Requested by ${request.name})</small>`;
 		if (request.comment) {
-			buf += Utils.html`<br /><p style="margin-left:5px;font-size:9pt;color:gray">${request.comment}</p>`;
+			buf += Utils.html`<br />${request.comment}</small></div>`;
 		} else {
-			buf += `</small></p>`;
+			buf += `</small></div>>`;
 		}
-		this.addBox(buf);
-		room.update();
+		room.add(`|c|${request.name}|/uhtml ${request.name},${buf}`);
```

Comment:
Why is this uhtml?

Reply:
Good question, I thought I told Mia to fix that. Or was that a different thing?